app/scripts/controllers/dashboard.coffee

```diff
@@ -98,12 +98,25 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
         watchCompute 'ptratioComputedMax',
           dependencies: ['allSchools']
           waitForPromise: true
-          computer: ([allSchools]) -> utils.max allSchools, 'PUPIL_TEACHER_RATIO', 10
+          computer: ([allSchools]) ->
+            max = utils.max allSchools, 'PUPIL_TEACHER_RATIO', 100
+            if max?
+              max.then (value) -> if value != $scope.range.ptratio.max
+                $scope.range.ptratio.max = value
```

Comment:
`computer` functions should be kept pure. We could move this check/assignment into a `$scope.$watch` function, or better, make a `watchCompute` for it?

(or, since it's working, we can just merge...)